### PR TITLE
tracing for pending_known map

### PR DESCRIPTION
**Pull request by rphmeier**

### CODE REVIEW

File: node/network/approval-distribution/src/lib.rs [hunk 1]
```diff
@@ -201,7 +201,17 @@ impl State {
...
+				self.pending_known.retain(|h, _| {
+					let stale = !view.contains(h);
+					if stale {
+						tracing::trace!(
+							target: LOG_TARGET,
+							"Cleaning up stale pending messages",
+						);
+					}
+
+					stale
```
Comment (ordian) [inline node/network/approval-distribution/src/lib.rs:213]: you mean `!stale`?

**Review by ordian — APPROVED** [review]

**Comment by rphmeier** [comment]
bot merge

**Review by drahnr — APPROVED** [review]

**Comment by ordian** [comment]
bot merge cancel

**Comment by rphmeier** [comment]
bot merge

**Pull request merged**